composables/usePageShow.ts

```
@@ -0,0 +1,7 @@
+export const usePageShow = async (fn: () => void | boolean | Promise<boolean>) => {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)